_modules/week-8.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Week 8 - How should Big Tech contribute to digital commons ?
+---
+
+Slides
+: **PDF**{: .label .label-purple }
+  : [Download](/assets/pdfs/Digital%20Commons%20and%20Public%20Policy%20-%20Week%201.pdf)
+
+Bibliography
+: **VIDEO**{: .label .label-purple }
+  : [Netflix Trailer](https://youtu.be/4RdncisZ_QA)
```

Comment:
Can you put the [actual bibliography](https://docs.google.com/document/d/1exXJnhWLa8VTamIMfb_uBwpK3RTVpnoQ1OYUsbd17f8/edit#heading=h.lxjdghsww83h) entries for week 8 please ?